client/driver/executor/executor_linux.go

```diff
@@ -499,3 +577,115 @@ func newLibcontainerConfig(command *ExecCommand) *lconfigs.Config {
 	configureCgroups(cfg, command)
 	return cfg
 }
+
+// configureResourceContainer
+func (e *UniversalExecutor) configureResourceContainer(pid int) error {
```

Comment:
Hmm I don't like mixing the universal and libcontainer code in one file

Reply:
Yeah I agree. `executor_universal_linux.go`?